webapp/channels/src/sass/components/_buttons.scss

```diff
@@ -298,6 +302,20 @@ button {
             }
         }
     }
+    &.btn-tertiary.btn-inverted {
```

Comment:
Should this also have disabled and danger variants as well?

Reply:
Yes i have added a ticket for UX to include these styles, i have added just whats required for the current pr